{{cookiecutter.app_name}}/assets/js/script.js

```diff
@@ -0,0 +1,3 @@
+(function myfunc($, window) { // eslint-disable-line no-unused-vars
```

Comment:
Now that we're using a module loader, `script.js` no longer needs an IIFE. How about we replace it with a file that just has a comment:

```js
// App initialization code goes here
```